zmon_aws_agent/main.py

```diff
@@ -147,6 +148,7 @@ def main():
         dynamodbs = aws.get_dynamodb_tables(region, infrastructure_account)
         certificates = aws.get_certificates(region, infrastructure_account)
         aws_limits = aws.get_limits(region, infrastructure_account, apps, elbs)
+        sqs = aws.get_sqs_queues(region, infrastructure_account)
```

Comment:
It is implemented in the ``get_running_apps()``

https://github.com/zalando-zmon/zmon-aws-agent/blob/master/zmon_aws_agent/aws.py#L178

The same logic can be applied here